test/Test/Cardano/Prelude/Helpers.hs

```diff
@@ -24,6 +26,17 @@ assertEitherIsRight func val = case func val of
   Left  err -> failWith Nothing (show $ sformat build err)
   Right _   -> success
 
+assertMaybeIsJust :: MonadTest m => (a -> Maybe b) -> a -> m ()
```

Comment:
Looks good, although I find the name slightly counterintuitive. I didn't expect it to be applying a function to the value. Can you think of a name that captures the application somehow?

Reply:
This naming aligns with `assertEitherIsLeft` etc. I think this name is fine and I really don't think I could come up with something better, but I would be happy to listen to suggestions from others.

Reply:
Maybe you think the type should be:
```
assertMaybeIsJust :: MonadTest m => Maybe a -> m ()
```
so it applies to a value rather than a function?

Reply:
Regarding the name, why not `assertIsJust` (I'm just curious) `assertMaybeIsJust` reads a bit like "assert that maybe is a just value" :P